photonlib-java-examples/src/main/java/org/photonlib/examples/simposeest/robot/Constants.java

```diff
@@ -94,9 +97,8 @@ public class Constants {
     public static final double kFarTgtXPos = Units.feetToMeters(54);
     public static final double kFarTgtYPos =
             Units.feetToMeters(27 / 2) - Units.inchesToMeters(43.75) - Units.inchesToMeters(48.0 / 2.0);
-    public static final Pose2d kFarTargetPose =
-            new Pose2d(new Translation2d(kFarTgtXPos, kFarTgtYPos), new Rotation2d(0.0));
+    public static final Pose3d kFarTargetPose = new Pose3d(new Translation3d(kFarTgtXPos, kFarTgtYPos, targetHeightAboveGround), new Rotation3d(0.0, 0.0, 0.0));
```

Comment:
`targetHeightAboveGround` at 81.19" is the height of the _bottom_ of the vision target. This doesn't seem like the right value for the pose. By default, shouldn't it be the _center_ of the vision target?
Remembering 2020, we had our Target Offset Point at the _top_ of the vision target so our goal would be the center of the Outer Port.

Reply:
Yes, fixed